boxes/config.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+boxes:
+  -
+    output_box_name: mysql
```

Comment:
Why is this named "output_box_name", is it a convention? For me, just having it as "box_name" feels natural.

WDYT?

Reply:
@msmshariq IMO the key name should indicate the intended purpose of it. Plus, we also have another configuration named, "base_box". I think the best key name to use here is 'output_box' rather than suggested 'box_name'. WDYT?